plugins/module_utils/statuscake.py

```diff
@@ -105,16 +105,18 @@ def create(self):
             if "check_rate" not in self.config:
                 self.config["check_rate"] = 300
             # Convert all _csv arguments to expect lists rather than strings
-            for key, value in self.config.items():
-                if key == "contact_groups":
-                    value = ",".join(value)
-                if key == "dns_ip":
-                    value = ",".join(value)
-                if key == "status_codes":
-                    value = ",".join(value)
-                if key == "tags":
-                    value = ",".join(value)
+            if "tags" in self.config:
```

Comment:
A few thoughts:
1. This needs to occur for create and update.
2. We need to add a new key for each of these. E.g. `tags` becomes `tags_csv`
3. We may want to consolidate this functionality into a prepare_data() method

Reply:
10-4, Mr. Copelco.